packages/runtime/runtime-utils/src/utils.ts

```diff
@@ -27,3 +28,43 @@ export async function seqFromTree(
 	const attrib = await readAndParseBlob<IDocumentAttributes>(attributesHash);
 	return attrib.sequenceNumber;
 }
+
+/**
+ * Encode compact ID (returned by IContainerRuntime.generateDocumentUniqueId()) to a compact string representation.
+ * While this is the main usage pattern, it works with any non-negative integer or a string.
+ * Strings are retured as is, and assumed to be UUIDs, i.e. unique enough to never overlap with
+ * numbers encoded as strings by this function. Any other strings are likely to run into collisions and should not be used!
+ * This function is useful in places where we serialize resulting ID as string and use them as strings, thus we are not
+ * gaining any efficiency from having a number type.
+ * We do not provide a decode function, so this API is only useful only result is stored and there is no need to go back to origianl form.
+ * @param numArg - input number, non-negative integer.
+ * @param prefix - optinal string prefix
+ * @returns A string - representation of an input
+ * @internal
+ */
+export function encodeCompactIdToString(numArg: number | string, prefix = "") {
+	if (typeof numArg === "string") {
+		return numArg;
+	}
+	// WARNING: result of this function are encodeCompactIdToStringn storage!
```

Comment:
> result of this function are encodeCompactIdToStringn storage!

What does this line mean?

Reply:
Do you mean "result of this function are encoded in storage"?

Reply:
"result of this function are stored in storage", Looks like rename across whole repot somehow captured wrong word :)